geopandas/tests/test_geocode.py

```diff
@@ -13,16 +14,14 @@
 from pandas.util.testing import assert_series_equal
 from geopandas.tests.util import mock, assert_geoseries_equal
 
+geopy = pytest.importorskip("geopy")
 
-def _skip_if_no_geopy():
-    try:
-        import geopy
-    except ImportError:
-        raise pytest.skip("Geopy not installed. Skipping tests.")
-    except SyntaxError:
-        raise pytest.skip("Geopy is known to be broken on Python 3.2. "
-                          "Skipping tests.")
+from geopy.exc import GeocoderNotFound
+from geopy.geocoders import GoogleV3
 
+@pytest.mark.skipif(
+    sys.version_info[0], sys.version_info[1]== (3,2),
+    reason="Geopy is known to be broken on Python 3.2.")
```

Comment:
This can be removed I think, we don't test or officially support python 3.2

Reply:
Ah ok, the original test had a 3.2 exception but I've removed it now!